README.md

```diff
@@ -258,14 +254,27 @@ When selecting which machines to deploy, the following constraints must be obser
 
 ## Resource Constraints
 
-Mesos agent nodes are where DC/OS services will be installed. Mesos will auto-detect the amount of resources available on these machines, with the following constraint:
+DC/OS services will be installed on Mesos agent nodes. Mesos will auto-detect the amount of resources available on these machines, with the following constraint:
 
 - Mesos reserves half or 1 GB of each machine's memory for overhead (whichever is least)
 
 For example, `m1` has 3328 MB memory by default. Some of that memory will be taken by OS and DC/OS component processes (~ MB). 1 GB will be reserved by Mesos as overhead. The rest will be offered to Mesos frameworks for launching tasks (~ MB).
 
 **IMPORTANT**: Make sure your local machine has enough memory to launch all your desired VMs, otherwise your machine may lock up as all the memory is consumed.
 
+# Destroy Your Cluster
+
+The following command destroys your cluster and any data stored there:
+
+`vagrant destroy -f`
+
+# Upgrade dcos-vagrant
```

Comment:
I see. So in this sentence, "To upgrade to a new version of docs-vagrant," you're talking about the repo?